### ISSUE TITLE
Update Windows container version compatibility for Server 2022

### ISSUE THREAD

**Issue by erenes** [MicrosoftDocs/Virtualization-Documentation#1632]
Windows server 2022 has been released, but the documentation about Windows container version compatibility has not been updated reflecting the new images/host OS. Based on previously released versions it seems to make sense that Windows Server 2022 will support any server 2019 image under hyper-v isolation, but the reverse is not true? 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c47037c4-4b2e-b41c-64a3-d82ec79c18c4
* Version Independent ID: 947b1d95-3cd5-53d4-042f-71afe8bb3bdf
* Content: [Windows Container Version Compatibility](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-20H2%2Cwindows-10-20H2)
* Content Source: [virtualization/windowscontainers/deploy-containers/version-compatibility.md](https://github.com/MicrosoftDocs/Virtualization-Documentation/blob/live/virtualization/windowscontainers/deploy-containers/version-compatibility.md)
* Product: **windows-server**
* Technology: **windows-containers**
* GitHub Login: @v-susbo
* Microsoft Alias: **v-susbo**

**Comment by v-susbo** [comment]
Yes, we will be updating this topic with WS 2022 information when the actual product release date occurs (soon).

**Comment by v-susbo** [comment]
These topics have now been updated: https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility?tabs=windows-server-2022%2Cwindows-10-20H2 and https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/base-image-lifecycle
